docs: redeem script clarification (#463)
BoltzExchange/boltz-backend: docs/api.md

Responses also contain one additional value:

> * `redeemScript`: redeem script from which the `address` is derived. The redeem script should be used, [to verify](dont-trust.-verify.md#utxo-chain-address-verification) that Boltz didn't provide a different address.

Reviewer: Different to what?
```suggestion
* `redeemScript`: redeem script from which the `address` is derived. The redeem script should be used, [to verify](dont-trust.-verify.md#utxo-chain-address-verification) that Boltz did provide the correct address.
```
